.github/workflows/unit-test.yml

```diff
@@ -38,5 +45,22 @@ jobs:
         with:
           go-version: "1.21"
 
+      - name: Setup jq
+        uses: dcarbone/install-jq-action@v2.0.2
+        with:
+          version: "${{ inputs.version }}"
+
       - name: run unit tests
-        run: GOPATH=$(go env GOPATH) make -j $(nproc) test
+        run: |
+          export ROOK_UNIT_JQ_PATH="$(which jq)"
+          GOPATH=$(go env GOPATH) make -j $(nproc) test | tee output.txt
```

Comment:
If the unit tests fail, this step will fail, right? Just wanted to double check that piping to `tee` doesn't mess up the failure code

Reply:
It woked